### PR TITLE
[Clean Up Portal Lib] LPS-145946 xalan.jar

### PR DESCRIPTION
**Pull request by dantewang**
@tinatian @shuyangzhou This has been reviewed by Forms team (in charge of DDM): https://github.com/liferay-forms/liferay-portal/pull/1391#issuecomment-1020108828

@vicnate5 @zhangyan0701 Please review the results once the tests are done, and trigger other batches that may be affected.

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:environments

**Comment by dantewang** [comment]
ci:test

**Comment by vicnate5** [comment]
@dantewang @tinatian No regressions found.

File: modules/third-party/org-outerj-daisy-diff/patches/LPS-199999.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+diff --git a/org/outerj/daisy/diff/helper/CleanBrokenAttributeQNamesHandler.java b/org/outerj/daisy/diff/helper/CleanBrokenAttributeQNamesHandler.java
```
Comment (tinatian) [inline modules/third-party/org-outerj-daisy-diff/patches/LPS-199999.patch:1]: @dantewang The file name LPS-199999.patch is wrong.

**Comment by tinatian** [comment]
@dantewang Please see my comment above and https://github.com/dantewang/liferay-portal/pull/2151.

Please fix it and resend.

Thanks.
Tina.

**Comment by tinatian** [comment]
ci:close

File: modules/third-party/org-outerj-daisy-diff/patches/LPS-199999.patch (hunk 1 above)
Comment (dantewang) [reply modules/third-party/org-outerj-daisy-diff/patches/LPS-199999.patch:1]: @tinatian Ah sorry I forgot to change it. Will resend.

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/266

**Pull request closed without merging**